SCClassLibrary/Common/Collections/ObjectTable.sc

```diff
@@ -38,7 +38,29 @@ TwoWayIdentityDictionary : Collection
 		^objToID.at(obj);
 	}
 
+	copy {
+		^this.class.prNew(idToObj.copy, objToID.copy)
+	}
+
+	storeOn { arg stream;
+		if (stream.atLimit) { ^this };
+		stream << this.class.name << ".prNew(";
+		[idToObj, objToID].storeItemsOn(stream);
```

Comment:
I don't think this is right. Its duplicate information. You should only have to store the Associations, and those should load into both tables when the object is created. 

TwoWayIdentityDictionary[ \one -> 1, \two -> 2] should work as a constructor

same with copy: you don't need to copy both dictionaries. they are just flops of each other, right ?